src/form/Form.tsx

```diff
@@ -166,6 +168,9 @@ function Form({
 
   const [repeats, setRepeats] = useState(0);
 
+  // Stripe - the card elements on the active step, if any
+  const [cardElementMappings, setCardElementMappings] = useState({});
```

Comment:
this state management, the setCardElement function below, and the setupPAymentMethod function initialization imo can all be condensed into a single usePayments custom hook

Reply:
Created the hook function usePayments.